From ceaa8c69f953f6c4608f8c702940e2c5423454c2 Mon Sep 17 00:00:00 2001
From: Ferry Toth <ftoth@exalondelft.nl>
Date: Tue, 29 Dec 2020 18:48:56 +0100
Subject: [PATCH] u-boot: Fix path to merge_config.sh

Current u-boot recipe in dunfell fails during do_configure because
it cannot find the merge_config.sh. Fix the path

Suggested-by: nishantpoorswani <nishantpoorswani@gmail.com>
Signed-off-by: Ferry Toth <ftoth@exalondelft.nl>
---
 meta/recipes-bsp/u-boot/u-boot.inc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meta/recipes-bsp/u-boot/u-boot.inc b/meta/recipes-bsp/u-boot/u-boot.inc
index a88a7a1120..a419e486c5 100644
--- a/meta/recipes-bsp/u-boot/u-boot.inc
+++ b/meta/recipes-bsp/u-boot/u-boot.inc
@@ -80,7 +80,7 @@ do_configure () {
                 if [ $j -eq $i ]; then
                     oe_runmake -C ${S} O=${B}/${config} ${config}
                     if [ -n "${@' '.join(find_cfgs(d))}" ]; then
-                        merge_config.sh -m -O ${B}/${config} ${B}/${config}/.config ${@" ".join(find_cfgs(d))}
+                        ${S}/scripts/kconfig/merge_config.sh -m -O ${B}/${config} ${B}/${config}/.config ${@" ".join(find_cfgs(d))}
                         oe_runmake -C ${S} O=${B}/${config} oldconfig
                     fi
                 fi
@@ -95,7 +95,7 @@ do_configure () {
         else
             oe_runmake -C ${S} O=${B} oldconfig
         fi
-        merge_config.sh -m .config ${@" ".join(find_cfgs(d))}
+        ${S}/scripts/kconfig/merge_config.sh -m .config ${@" ".join(find_cfgs(d))}
         cml1_do_configure
     fi
 }
-- 
2.27.0

